### PR TITLE
tweak cluster role for kubeless ui

### PR DESCRIPTION
**Pull request by sebgoa**
otherwise we cannot get the logs of the function and we cannot call them

### CODE REVIEW

**Comment by sebgoa** [comment]
@anguslees and that one too while you are at it

File: kubeless-ui.jsonnet [hunk 1]
```diff
@@ -12,7 +12,17 @@ local host = "kubeless-ui";
...
+        verbs: ["get", "list", "watch", "create", "patch", "delete"],
+      },
+      {
+        apiGroups: [""],
+        resources: ["pods","pods/log"],
+        verbs: ["get", "list"],
+      },
+      {
+        apiGroups: [""],
+        resources: ["services","services/proxy],
```
Comment (sebgoa) [inline kubeless-ui.jsonnet]: ah geez..."

**Comment by sebgoa** [comment]
Ok, I am going to merge this as well...you can hit me with a stick later.

**Comment by anguslees** [comment]
(lgtm)

**Pull request merged**